docs/core/component.md

```diff
@@ -508,8 +508,8 @@ AFRAME.registerComponent('c', {});
 
 [sound]: ../components/sound.md
 
-`multiple` allows for a component to have multiple instances. By default, since
-`multiple` is set to `false`, a component could have one instance. For
+`multiple` allows for a component to have multiple instances per entities. By default, since
```

Comment:
Maybe simpler this way? `multiple allows multiple instances of the component on an entity`